src/crimson/os/seastore/cache.h

```diff
@@ -372,13 +372,38 @@ class Cache {
    *
    * Allocates a fresh extent.  addr will be relative until commit.
    */
```

Comment:
This and other bits of this patch get significantly reworked by 6728982087afe9157a7318b11dc2f6b7ab2dc40f (crimson/os/seastore: make fresh extents go though the extent placement manager).  Please move any overlapping changes from that patch back into this one to avoid needing a review on both versions.

Reply:
This was meant to be a comment about e2d7e41b5849e6782085143898d704ddabe74db7 (crimson/os/seastore: add extent placement manager).

Reply:
ok:-)